forge-gui/res/cardsfolder/upcoming/flitting_guerrilla.txt

```diff
@@ -0,0 +1,13 @@
+Name:Flitting Guerrilla
+ManaCost:2 B
+Types:Creature Faerie Rogue
+PT:2/2
+K:Flying
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigMill | TriggerDescription$ When CARDNAME dies, each player mills two cards. Then you may exile CARDNAME. When you do, put target creature or battle card from your graveyard on top of your library. (To mill two cards, a player puts the top two cards of their library into their graveyard.)
+SVar:TrigMill:DB$ Mill | Defined$ Player | NumCards$ 2 | SubAbility$ DBRemember
+SVar:DBRemember:DB$ Pump | RememberObjects$ TriggeredNewCardLKICopy | SubAbility$ DBImmediateTrig
```

Comment:
https://github.com/Card-Forge/forge/blob/96ec9d201241ad050c2f0048f39f596a06089b9f/forge-game/src/main/java/forge/game/card/CardProperty.java#L1964

Insert getRootAbility() call there + for NotTriggered then no Remember needed